LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2017 Oscar
+Copyright (c) 2017-2020 Oscar Bartra Carreras <obartra@gmail.com>
+Copyright (c) 2020 Dan Weber <dan@notatypical.agency>
```

Comment:
This is the part I want to be careful about, I'm more than happy to give credit where credit is due, but ownership needs to remain exclusively for the project

Reply:
Check the changes and see if that fits your criteria.  

After this we should make one last minor or patch version for SSIM.js with the addition of weberSsim.  Then a new major/minor version or such with the breaking change for default.

Reply:
The issue is with ownership, note that this is true of any other open source project (e.g. if you contribute to react you may get your name added to contributors list but you don't get any claims of ownership / copyright over the react code, even the lines you contributed).

This is the same here, although you've come up with a really fast approach (it may be worth publishing it as an improvement to SSIM's a bezkrovny's), I don't want to add any ownership claims in the repository about the code / implementation.

That is to say while you will get credit on the work (the algorithm already has your name, we agreed to add it to the README and to the docs), the implementation is open source and I want to make sure there are no licensing issues (real or perceived) stemming from this.

Reply:
Mmm... If you're concerned about licensing and ownership rights, I think you should read this --
https://opensource.google/docs/cla/ -- Google's Contributor License Agreement and probably install a CLA of your own.

In this case, I don't think you should have any concerns.  I have the right and authority to make these submissions.  Any changes, works of original authorship, derivative works, or other contributions I make to the SSIM project (this project) are licensed under the MIT license and I expressly grant permission to you under this license.  This license grants you a perpetual, irrevocable, right to use the work for any purpose so long as you, your heirs, beneficiaries, customers, and any other unnamed person don't hold me or any other person or entity related to me liable in any way whatsoever.  If you'd like you can print this out, send it to me, and I'll sign it, so you can keep it on file.

Reply:
A CLA is a good thing for this project anyway.  I'm going to file a ticket so we can set it up across the board.

Reply:
Ok I've set up the CLI, if that sounds good to you, let's remove the new MIT license for the Weber algorithm, sign the the CLA and we are good to go